lib/models/project.js

```diff
@@ -530,6 +530,32 @@ Project.prototype.findAddonByName = function(name) {
 };
 
 /**
+  Generate test file contents.
+
+  This method is supposed to be overwritten by test framework addons
+  like `ember-cli-qunit` and `ember-cli-mocha`.
+
+  @protected
```

Comment:
ya, its basically public, and intended to be optionally overiden by a subclass, so public makes sense.